README.md

```diff
@@ -16,8 +16,13 @@ Instructions are as follows. Clone this repository:
 ```bash
 git clone https://github.com/hackersatcambridge/hac-website.git
 ```
+Navigate to the hac-website directory that you've now cloned. Create a file named `.env` and give it the following contents:
+```
+DATA_DIR=/root/hac-website/Data
```

Comment:
👍 Will move this to the `docker-compose` but since that is loaded before the `.env`, the `.env` will overwrite the docker-compose so we can have to have a think later about how to support multiple environments here